src/parser/signup.test.js

```diff
@@ -13,6 +13,6 @@ describe('signup parameters parser 📝', () => {
   test('It properly parses and returns the parameters for signup', () =>
     expect(parseSignup('!ac signup foo github/foo')).toEqual([
       'foo',
-      ['github/foo'],
+      ['github/foo', 'discord/undefined'],
```

Comment:
This is not tested properly. :) We should validate that is as long as a Discord id is, and that we don't have any letters or any extraneous things on the id.